workspace.dsl

```diff
@@ -0,0 +1,167 @@
+workspace "Appeal service" {
+	model {
+		properties {
+			"structurizr.groupSeparator" "/"
+		}
+
+		userLocalPlanningAuthority = person "Local Planning Authority" "A local authority issuing decisions on planning applications"
+		userAppellant = person "Appellant" "A member of the public appealing against a planning decision"
+
+		group "Planning Inspectorate" {
+			userCaseOfficer = person "Case officer" "Checks LPA questionnaire and uploads statements and comments"
+			userInspector = person "Inspector" "Visits appeal site and issues appeal decision"
+
+			systemAppsFo = softwareSystem "Appeals Front-Office" "Public dashboards for appellants and Local Planning Authorities" "Node.js, Azure Web App, Azure Function App" {
+				tags "InternalCollaborationSystem"
+
+				containerFoFileStorage = container "File storage" "Stores all files uploaded in the Front-Office"
+			}
+
+			systemAppsBo = softwareSystem "Appeals Back-Office" "Internally facing service allowing for management of planning appeals in England"  {
+				tags "InternalCollaborationSystem"
+
+				group "Web" {
+
+    				containerBoWeb = container "Back-Office Web" "Back-office website" "Node.js, Azure Web App" {
+    					tags "Microsoft Azure - App Services - Web"
+    				}
+
+    				containerBoRedis = container "Redis" "Distributed session storage"
+    				containerBoAzureActiveDirectory = container "Entra ID" "Stores login details and assigned user groups"
+    				containerBoFileStorage = container "File storage" "Stores all files related to appeals"
```

Comment:
As per the Azure theme comment, can use a tag of `Microsoft Azure - Storage Container` here (or whatever is most appropriate)